chef/cookbooks/barclamp/spec/libraries/barclamp_library_spec.rb

```diff
@@ -0,0 +1,64 @@
+require "spec_helper"
+
+require_relative "../../libraries/barclamp_library"
+
+describe BarclampLibrary::Barclamp::Inventory::Disk do
+
+  before(:each) do
+    @chef_run = ::ChefSpec::Runner.new
+    @node = @chef_run.node
+    @node.default[:block_device] = {
+      dm0: { removable: "0" },
+      xvd1: { removable: "0" },
+      xvd2: { removable: "1" }
+    }
+  end
+
+  it "unclaimed method returns the proper number of unclaimed devices" do
```

Comment:
Specification header blocks are supposed to read like English, but the `it` doesn't make sense here.  Also `unclaimed method` can be abbreviated to `#unclaimed`:

    specify "#unclaimed returns the proper number of unclaimed devices" do